server/models/models.js

```diff
@@ -11,25 +11,29 @@ let playerSchema = mongoose.Schema({
     Games: Number
 });
 
+let teamSchema = mongooose.Schema({
+    Owner: { type: Schema.Types.ObjectId, ref: 'User' },
```

Comment:
your naming convention is wrong. see https://www.w3schools.com/js/js_conventions.asp for advice on properly naming your properties and variables.